Tools/autotest/vehicle_test_suite.py

```diff
@@ -4632,11 +4632,17 @@ def install_example_script_context(self, scriptname):
         self.install_example_script(scriptname)
         self.context_get().installed_scripts.append(scriptname)
 
-    def install_test_script_context(self, scriptname):
+    def install_test_script_context(self, scriptnames):
         '''installs an test script which will be removed when the context goes
         away'''
-        self.install_test_script(scriptname)
-        self.context_get().installed_scripts.append(scriptname)
+        if getattr(scriptnames, '__iter__', None) is None:
+            scriptname = [scriptnames]
+        for scriptname in scriptnames:
+            self.install_test_script(scriptname)
+            self.context_get().installed_scripts.append(scriptname)
```

Comment:
Strings have an `__iter__` method too, this will install every letter as a separate string. The test should probably be `if isinstance(scriptnames, str)`.

Reply:
The assignment is also incorrect.

Reply:
> Strings have an `__iter__` method too, this will install every letter as a separate string. The test should probably be `if isinstance(scriptnames, str)`.

Fixed, thanks!

Reply:
> The assignment is also incorrect.

Wow, really needed extra coffee.

Probably shouldn't have removed the only caller to the singular-script case....